### PR TITLE
Update Uganda config to match VHT template

### PR DESCRIPTION
**Pull request by kennsippell**
#39

### CODE REVIEW

**Comment by kennsippell** [comment]
> Can we add the [training instance](https://moh-ug-test.dev.medicmobile.org/) and [dev instance](https://vhtapp-4x.dev.medicmobile.org/) for testing and demo purposes for our thursday call

**Comment by jonathanbataire** [comment]
Hey @kennsippell  had a look at this 
- auto-generated (village) works fine but i was hoping we could eliminate VHT Area name all together so that the entire thing is auto generated from vht name and village
we could show the auto generated name as a string (maybe add an edit button)
- Sex should be a drop down with just male and female

**Comment by kennsippell** [comment]
> could eliminate VHT Area name all together so that the entire thing is auto generated from vht name and village

Going to handle this via https://github.com/medic/cht-user-management/issues/47 since it affects both UG and KE. Can we live with it? 

> Sex should be a drop down with just male and female

Tracking via https://github.com/medic/cht-user-management/issues/55. This one is pretty easy and I might do it in this PR.

File: src/config/chis-ug/config.json [hunk 1]
```diff
@@ -2,16 +2,21 @@
 {
   "domains": [
     {
-      "friendly": "UG VHT",
+      "friendly": "eCHIS Uganda",
       "domain": "echis.health.go.ug"
+    },
+    {
+      "friendly": "eCHIS Uganda Staging",
+      "domain": "vhtapp.dev.medicmobile.org"
```
[the remark this answers is not in the document]
Comment (kennsippell) [reply src/config/chis-ug/config.json]: Updated

**Comment by kennsippell** [comment]
@jonathanbataire It's going to take quite a few PRs to get this ready for Uganda. It's my preference that we merge small changes frequently and move toward readiness.  Would you be available to review these changes?

**Review by jonathanbataire — APPROVED** [review]
LGTM

**Pull request merged**